proto/cmp/services/claim/v1/claim_settlement_decision.proto

```diff
@@ -0,0 +1,48 @@
+syntax = "proto3";
+
+package cmp.services.claim.v1;
+
+import "cmp/services/claim/v1/claim_types.proto";
+import "cmp/types/v1/common.proto";
+import "cmp/types/v2/price.proto";
+import "google/protobuf/timestamp.proto";
+
+message ClaimSettlementDecisionRequest {
+  // Message header
+  cmp.types.v1.RequestHeader header = 1;
+
+  // Only respond with the products that are new, modified or deactivated after this
+  // timestamp.
+  //
+  // Timestamps may be used for both off-chain and on-chain operations.
+  // For on-chain operations, only seconds are supported, and nanoseconds
+  // will be ignored.
+  google.protobuf.Timestamp modified_after = 2;
+
+  // Claim ID: Submission ID, Processing ID and Carrier ID
+  cmp.services.claim.v1.ClaimId claim_id = 3;
+
+  // Claim Status
+  cmp.services.claim.v1.ClaimStatus status = 4;
+
+  // TODO: Is this reference applied for Acceptance and Rejection ?
+  string decision_reference = 5;
+
+  // More info about decision such as rejection reason.
+  string decision_detail = 6;
+
+  // Pay-out transaction
+  cmp.services.claim.v1.PayOutTransaction payout_transaction = 7;
+
+  // TODO: ask whether there could be multiple payouts for the same claim.
+  // Amount
+  cmp.types.v2.PriceDetail amount = 8;
```

Comment:
```suggestion
  // TODO: ask whether there could be multiple payouts for the same claim.
  //
  // Amount
  cmp.types.v2.PriceDetail amount = 8;
```
Markdown renders these as a single paragraph. So, if we don't add an additional new line between them the "Amount" will be rendered just after the "... for the same claim."